update_gh_pages.sh

```diff
@@ -1,10 +1,37 @@
-#!/bin/bash
+#!/bin/sh
+
+# Make sure we're not echoing any sensitive data
+set +x
```

Comment:
Are you sure that this is not exposing environment variables? I believe `-x` exposes all variable content:

```
: tmp $ cat foo.sh 
#! /bin/sh

set -x

touch $X
: tmp $ env X=bar ./foo.sh 
+ touch bar
```

Reply:
Yes, and I think the `+` in `set +x`  disables this `-x` flag.